test/tools/unit_test_runner.d

```diff
@@ -336,5 +336,6 @@ int main(string[] args)
     buildStrtold();
     execute(dmdPath, "@" ~ cmdfilePath);
 
+    stderr.writefln("[unit] Starting to run %s test files", nrOfFiles);
     return spawnProcess(outputPath).wait();
```

Comment:
We should probably also prepend the output with a prefix here as it currently looks like this:

```
[unit] Starting to build all test files
 ... runnable/test9271.d            -fPIC ()
 ... runnable/printargs.d           -fPIC ()
...
 ... runnable/testfile.d            -fPIC ()
 ... runnable/b18034.d              -O -fPIC (-inline -release -g)
 ... runnable/test_dip1006.d        -check=in=off -check=out=off -check=invariant=off -fPIC ()
[unit] Starting to run all test files
1 tests, 0 failures
 ... runnable/mod1.d                -fPIC ()
 ... runnable/test15862.d           -O -release -fPIC ()
```

Reply:
Isn’t that what your change is doing?

Reply:
Nope. The log is already from this change. We probably need to create our own `Pipe` and do sth. like:

```d
while (!pid.tryWait.terminated) {
  // if pipe has new data, read it from the buffer, split by line and append `[unit]`
  sleep(1.msecs);
}
```

Reply:
Sorry, I'm not following.

Reply:
`spawnProcess` directly forwards all stdout, stderr etc. We can't add `[unit]` prefix to the lines. This change is only added it to the log commands, not things like:

```
1 tests, 0 failures
```

Though for the user to know where the message came from imho it probably would be better to at least do sth. like:

```
[unit] 1 tests, 0 failures
```

For this we need to either add the prefix in the actual runner or e.g. read line-wise as discussed above from the stdout pipe of the spawned process.

Reply:
Ah, ok, now I get it.